department-of-veterans-affairs/va.gov-cms
# VACMS-10099: Place Q&As Modal in the CMS is not available with keyboard

## Description

Closes #10099.

## Testing done
Tested on my local.

## QA steps

1. Go to an FAQ page within the CMS. 
    https://staging.cms.va.gov/node/8233/edit
2. Scroll down to Q&A Groups
3. Use the keyboard to tab to the "Place Q&As" button
4. Hit enter to activate the button and activate the modal
5. Hit tab to move through the modal
6. The focus indicator should move back to the X
7. Once the X int he top right corner of the modal is back in focus hit neter to exit the modal.

### Definition of Done


### Select Team for PR review
